recipes/gperftools/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
```

Comment:
You need to add a `recipes/gperftools/config.yml` file that acts as an index.
See e.g. https://github.com/conan-io/conan-center-index/blob/master/recipes/7zip/config.yml

Reply:
thanks @madebr